### PR TITLE
Portal app: overview page first pass

### PR DESCRIPTION
**Pull request by rorysaur**
+ Portal app: overview page. This is the logged-in homepage of the wallet/portal, at `/assets/overview`. All links are stubs currently, and styling is unfinished.
+ Adds Kernel colors to the app's tailwind config.
+ Small component architecture changes, including renaming JSX files from `.js` to `.jsx`, and adding an app-wide Page component that contains the navbar, sidebar, and footer.

Screen Shot 2022-04-24 at 7 39 03 PM

### CODE REVIEW

File: packages/wallet/src/views/Assets.jsx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    title: `Contracts`,
+    description: `Explore all the best contracts`,
+    url: '/contracts',
+  },
+  {
+    title: `Develop`,
+    description: `Write your own code!`,
+    url: '/develop',
+  },
+]
```
Comment (rorysaur) [inline packages/wallet/src/views/Assets.jsx:67]: I know I added an App.config, but I feel like this content makes sense in this component's file, so you can change it all in one place.

**Comment by simonkernel** [comment]
Thanks @rorysaur. The build failure is my bad.

We have to patch a couple libraries we're using for the Projects app to make them compatible with the build step and I did not add it to the testing workflow.

Can you update your branch and pull in the last commit to see if this fixes it?

Thanks!

File: packages/wallet/src/App.js [hunk 2]
```diff
@@ -9,15 +9,15 @@
...
+import NotFound from 'views/NotFound.jsx'
 import 'App.css'
 
-const Wallet = lazy(() => import('views/Wallet.js'))
-const Assets = lazy(() => import('views/Assets.js'))
-const Register = lazy(() => import('views/Register.js'))
-const Create = lazy(() => import('components/Create.js'))
-const Import = lazy(() => import('components/Import.js'))
-const Auth = lazy(() => import('components/Auth.js'))
+const Wallet = lazy(() => import('views/Wallet.jsx'))
```
Comment (simonkernel) [inline packages/wallet/src/App.js:15]: nit: can we just leave out the file extension instead?

**Review by simonkernel — APPROVED** [review]
LGTM

**Pull request merged**